index.html

```diff
@@ -4064,10 +4068,10 @@ <h2>Elements</h2>
 <p>An ECMAScript <a>Object</a> <dfn>represents a web element</dfn>
  if it has a <a>web element identifier</a> <a>own property</a>.
 
-<p>Each <a>browsing context</a> has an associated <dfn>list of
- known elements</dfn>.
- When the <a>browsing context</a> is <a>discarded</a>,
- the <a>list of known elements</a> is discarded along with it.
+<p>Each <a>browsing context</a> has an associated <dfn>list of known
+ elements</dfn>.  When the <a>browsing
+ context</a>'s <a>navigable</a>'s <a>active window</a> is destroyed,
+ the <a>list of known elements</a> is destroyed along with it.
```

Comment:
Aha. So the list is still in the browsing context, but the lifetime is limited by the navigable. I assume this is done to avoid changing references in the spec to "browsing context's navigable". SGTM

NIT: should we say "list of known elements is **_cleared_**" instead of "**_destroyed_**"?

Reply:
This is going to go away with the Node cache changes, so I'm not going to touch it more here.